packages/inference/test/HfInference.test.ts

```diff
@@ -159,7 +172,7 @@ describe("HfInference", () => {
 					sequence:
 						"Hi, I recently bought a device from your company but it is not working as advertised and I would like to get reimbursed!",
 					labels: ["refund", "faq", "legal"],
-					scores: [0.877787709236145, 0.10522633045911789, 0.01698593981564045],
+					scores: [0.8777875304222107, 0.10522652417421341, 0.01698593609035015],
```

Comment:
Oh nice fix. It was an oversight on my behalf but these tests needs to simply check for the existence of an array with numbers. Checking for exact results will end up with test failures.

Reply:
just for context @Quatton I've fixed this issue on #31 using `expect.closeTo` but I guess `expect.any(Number)` would also also work